packages/zosfiles/src/methods/upload/Upload.ts

```diff
@@ -487,6 +488,12 @@ export class Upload {
             apiResponse.etag = uploadRequest.response.headers.etag;
         }
 
+        if (options.encoding != null) {
+            await Utilities.chtag(session, origUssname, Tag.TEXT, options.encoding);
```

Comment:
it seems that this line is not covered?

Do we not have any tests that call the `bufferToUssFile` with no encoding? 🤯 
If not, maybe we should modify/add one 😋

Reply:
Hey @jace-roell,
was this resolved in some un-pushed commit?